contents/blog/5-ways-to-improve-analytics-data.md

```diff
@@ -60,30 +60,24 @@ Now, you must remember all these differences when looking at your data! Annoying
   ```
 
 - For event names, use the `category:object_action` framework:
-
   - `category` describes the context the event took place in – e.g., `account_settings` or `signup_flow` 
-
   - `object` is a noun that refers to the component or location the event took place in, like `forget_password_button` or `pricing_page`.
-
   - `action` is a verb that describes what happened, like `click`, `submitted`, or `create`
-
   - Putting all of these together, you get `account_settings:forget_password_button_click` or `signup_flow:pricing_page_view` – The order is important as it makes it easier to find a specific event within a long list.
-
 - For property names:
-
   - Use the `object_adjective` pattern. For instance: `user_id`, `item_price`, or `member_count`.
-
   - Use `is/has` prefixes for boolean properties. For example: `is_subscribed`, `has_seen_upsell`.
-
   - If the property's value is a date or timestamp, include _date or _timestamp at the end. For example, `user_creation_date`, `last_login_timestamp`.
 
+> **Tip:** If you're tracking any events that are triggered by the system and not a user, such as automatically importing something on a regular basis, ensure you're marking these in a way that you can filter out later so they don't skew your active user metrics.
```

Comment:
not needed in PostHog :) so will exclude in order to prevent confusion with our users